### ISSUE TITLE
Not all query types supported

### ISSUE THREAD

**Issue by RonaldBarnes** [RonaldBarnes/dns-my-ip-addr-ad-blocker#4]
Not all query types are supported; some give invalid responses with broken headers, etc.
```
qtypeCodes = {
    1: "A",
    2: "NS",
    5: "CNAME",
    6: "SOA",
   10: 'NULL',
   11: 'WKS',
   12: "PTR",
   13: "HINFO",
   15: "MX",
   16: "TXT",
   33: 'SRV',
  252: 'AFXR',
  255: 'ANY'
}
```

Only types A, TXT, and MX return valid responses.

Type ANY doesn't even register a connection.

**Comment by RonaldBarnes** [comment]
While type ANY not connecting has been looked into - it's a strange one - CloudFlare was deprecating it and currently returns NOTIMP (NOT IMPlemented) on their DNS server at 1.1.1.1 for type ANY.

Google does support it at 8.8.8.8.

CloudFlare raised this prior to entirely phasing it out:

> 'Type ANY not supported   See draft-ietf-dnsop-refuse-any'

**Comment by RonaldBarnes** [comment]
The only type that makes sense whether querying "my.ip" or if a domain is NXDOMAIN, is type "A" - the default.

Type MX was implemented as a fun project but adds no real functionality.

The other types also provide no functionality, make no particular sense in the usage this tool was designed for, and unless someone want to sponsor the development, won't be implemented.
